### ISSUE TITLE
An expression with a function doesn't rewrite as expected

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo-phi-normalizer#553]
I expect this one:

```
{⟦ m ↦ ⟦ x ↦ ⟦ λ ⤍ Fn ⟧.ρ.k, k ↦ ⟦ Δ ⤍ 42- ⟧ ⟧.x ⟧}
```

to rewrite to this one:

```
{⟦ m ↦ ⟦ λ ⤍ Fn ⟧.ρ.k(ρ ↦ ⟦ x ↦ ⟦ λ ⤍ Fn ⟧.ρ.k, k ↦ ⟦ Δ ⤍ 42- ⟧ ⟧ ) ⟧}
```

However, something else is produced.

**Comment by yegor256** [comment]
@Anatolay please, check

**Comment by yegor256** [comment]
@Anatolay I suggest closing this issue when the problem is resolved in a released version of the normalizer. Until then, we can keep it open, thus reminding us that the problem is not yet solved (for me).

**Comment by 0crat** [comment]
@yegor256 Thank you for your contribution! While this ticket wasn't accepted, your effort is appreciated. You've earned -16 points, bringing your balance to [-78](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep in mind that every report helps improve our project. Stay motivated and [continue reporting](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) potential issues—your insights are valuable! Remember, [bugs are welcome](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) as they help us enhance our software quality. 🐞

**Comment by deemp** [comment]
@yegor256, this issue should be resolved in the v2.2.1. Please check it.

**Comment by yegor256** [comment]
@deemp perfect now, thanks!
